components/apimgt/org.wso2.carbon.apimgt.persistence/src/main/java/org/wso2/carbon/apimgt/persistence/APIConstants.java

```diff
@@ -51,6 +51,7 @@ public final class APIConstants {
     public static final String API_OVERVIEW_BUSS_OWNER_EMAIL = "overview_businessOwnerEmail";
     public static final String API_OVERVIEW_VISIBILITY = "overview_visibility";
     public static final String API_OVERVIEW_VISIBLE_ROLES = "overview_visibleRoles";
+    public static final String API_OVERVIEW_VISIBLE_ORGANIZATIONS = "overview_visibleOrganizations";
```

Comment:
are we setting organization visibility info to rxt? are we not setting it as a resource property

Reply:
fixed via https://github.com/wso2/carbon-apimgt/pull/12717/commits/d9dfd8d24eb52356e43f59579e416de2b29f6591